<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PlayerStatus</class>
 <widget class="QWidget" name="PlayerStatus">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>主角信息</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <property name="spacing">
    <number>0</number>
   </property>
   <item row="0" column="0">
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tab">
      <attribute name="title">
       <string>基本信息</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="0" column="0">
        <widget class="QLabel" name="name">
         <property name="text">
          <string>姓名:十个字符应该够用了吧</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLabel" name="NickName">
         <property name="text">
          <string>昵称:十个字符应该够用了吧</string>
         </property>
        </widget>
       </item>
       <item row="0" column="2">
        <widget class="QLabel" name="claimed">
         <property name="text">
          <string>自称:十个字符应该够用了吧</string>
         </property>
        </widget>
       </item>
       <item row="0" column="3">
        <widget class="QWidget" name="staminaBarWidget" native="true">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>0</height>
          </size>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <property name="spacing">
           <number>0</number>
          </property>
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item>
           <widget class="QLabel" name="staminaBarLabel">
            <property name="minimumSize">
             <size>
              <width>30</width>
              <height>20</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>30</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="text">
             <string>体力:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="MYStrip" name="staminaBarStrip" native="true">
            <property name="minimumSize">
             <size>
              <width>120</width>
              <height>20</height>
             </size>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="staminaBarNumberLabel">
            <property name="minimumSize">
             <size>
              <width>90</width>
              <height>20</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>90</width>
              <height>19</height>
             </size>
            </property>
            <property name="text">
             <string>999999/999999</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="Major">
         <property name="text">
          <string>专业:我一共有十三个字符,够长吧.</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1" colspan="2">
        <widget class="QLabel" name="Occupation">
         <property name="text">
          <string>职业:十个字符应该够用了吧!十个字符应该够用了吧!</string>
         </property>
        </widget>
       </item>
       <item row="1" column="3">
        <widget class="QWidget" name="energyBarWidget" native="true">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>0</height>
          </size>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <property name="spacing">
           <number>0</number>
          </property>
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item>
           <widget class="QLabel" name="energyBarLabel">
            <property name="minimumSize">
             <size>
              <width>30</width>
              <height>20</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>30</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="text">
             <string>精力:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="MYStrip" name="energyBarStrip" native="true">
            <property name="minimumSize">
             <size>
              <width>120</width>
              <height>20</height>
             </size>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="energyBarNumberLabel">
            <property name="minimumSize">
             <size>
              <width>90</width>
              <height>20</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>90</width>
              <height>19</height>
             </size>
            </property>
            <property name="text">
             <string>999999/999999</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="birthday">
         <property name="text">
          <string>生日:3000年10月10日</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QLabel" name="height">
         <property name="text">
          <string>身高:幼小</string>
         </property>
        </widget>
       </item>
       <item row="2" column="3">
        <widget class="QLabel" name="weight">
         <property name="text">
          <string>体重:普通的</string>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QWidget" name="hairColor" native="true">
         <layout class="QHBoxLayout" name="horizontalLayout_14">
          <property name="spacing">
           <number>0</number>
          </property>
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item>
           <widget class="QLabel" name="hairColorLable">
            <property name="font">
             <font>
              <pointsize>10</pointsize>
             </font>
            </property>
            <property name="text">
             <string>发色:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="ColorView" name="hairColorView" native="true">
            <property name="minimumSize">
             <size>
              <width>30</width>
              <height>30</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>30</width>
              <height>30</height>
             </size>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="QWidget" name="EyeColors" native="true">
         <layout class="QHBoxLayout" name="horizontalLayout_18">
          <property name="spacing">
           <number>0</number>
          </property>
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item>
           <widget class="QLabel" name="eyeColorsLable">
            <property name="font">
             <font>
              <pointsize>10</pointsize>
             </font>
            </property>
            <property name="toolTip">
             <string/>
            </property>
            <property name="text">
             <string>瞳色:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="LEyeColorsLable">
            <property name="font">
             <font>
              <pointsize>10</pointsize>
             </font>
            </property>
            <property name="toolTip">
             <string/>
            </property>
            <property name="text">
             <string>左</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="ColorView" name="LEyeColorsView" native="true">
            <property name="minimumSize">
             <size>
              <width>30</width>
              <height>30</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>30</width>
              <height>30</height>
             </size>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="REyeColorsLable">
            <property name="font">
             <font>
              <pointsize>10</pointsize>
             </font>
            </property>
            <property name="toolTip">
             <string/>
            </property>
            <property name="text">
             <string>右</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="ColorView" name="REyeColorsView" native="true">
            <property name="minimumSize">
             <size>
              <width>30</width>
              <height>30</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>30</width>
              <height>30</height>
             </size>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item row="3" column="3">
        <widget class="QWidget" name="skinColor" native="true">
         <layout class="QHBoxLayout" name="horizontalLayout_15">
          <property name="spacing">
           <number>0</number>
          </property>
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item>
           <widget class="QLabel" name="skinColorLable">
            <property name="font">
             <font>
              <pointsize>10</pointsize>
             </font>
            </property>
            <property name="text">
             <string>肤色:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="ColorView" name="skinColorView" native="true">
            <property name="minimumSize">
             <size>
              <width>30</width>
              <height>30</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>30</width>
              <height>30</height>
             </size>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item row="4" column="0" colspan="4">
        <widget class="QWidget" name="willpower" native="true">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>0</height>
          </size>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_4">
          <property name="spacing">
           <number>0</number>
          </property>
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item>
           <widget class="QLabel" name="willpowerLabel">
            <property name="minimumSize">
             <size>
              <width>30</width>
              <height>20</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>30</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="text">
             <string>意志:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="MYStrip" name="willpowerStrip" native="true">
            <property name="minimumSize">
             <size>
              <width>120</width>
              <height>20</height>
             </size>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="willpowerBarNumberLabel">
            <property name="minimumSize">
             <size>
              <width>90</width>
              <height>20</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>90</width>
              <height>19</height>
             </size>
            </property>
            <property name="text">
             <string>999999/999999</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item row="5" column="0" colspan="4">
        <widget class="QWidget" name="quick" native="true">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>0</height>
          </size>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_5">
          <property name="spacing">
           <number>0</number>
          </property>
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item>
           <widget class="QLabel" name="quickLabel">
            <property name="minimumSize">
             <size>
              <width>30</width>
              <height>20</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>30</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="text">
             <string>敏捷:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="MYStrip" name="quickStrip" native="true">
            <property name="minimumSize">
             <size>
              <width>120</width>
              <height>20</height>
             </size>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="quickBarNumberLabel">
            <property name="minimumSize">
             <size>
              <width>90</width>
              <height>20</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>90</width>
              <height>19</height>
             </size>
            </property>
            <property name="text">
             <string>999999/999999</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item row="6" column="0" colspan="4">
        <widget class="QWidget" name="physique" native="true">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>0</height>
          </size>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_6">
          <property name="spacing">
           <number>0</number>
          </property>
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item>
           <widget class="QLabel" name="physiqueLabel">
            <property name="minimumSize">
             <size>
              <width>30</width>
              <height>20</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>30</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="text">
             <string>体质:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="MYStrip" name="physiqueStrip" native="true">
            <property name="minimumSize">
             <size>
              <width>120</width>
              <height>20</height>
             </size>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="physiqueBarNumberLabel">
            <property name="minimumSize">
             <size>
              <width>90</width>
              <height>20</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>90</width>
              <height>19</height>
             </size>
            </property>
            <property name="text">
             <string>999999/999999</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item row="7" column="0" colspan="4">
        <widget class="QWidget" name="charm" native="true">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>0</height>
          </size>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_7">
          <property name="spacing">
           <number>0</number>
          </property>
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item>
           <widget class="QLabel" name="charmLabel">
            <property name="minimumSize">
             <size>
              <width>30</width>
              <height>20</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>30</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="text">
             <string>魅力:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="MYStrip" name="charmStrip" native="true">
            <property name="minimumSize">
             <size>
              <width>120</width>
              <height>20</height>
             </size>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="charmBarNumberLabel">
            <property name="minimumSize">
             <size>
              <width>90</width>
              <height>20</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>90</width>
              <height>19</height>
             </size>
            </property>
            <property name="text">
             <string>999999/999999</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item row="8" column="0" colspan="4">
        <widget class="QWidget" name="intelligence" native="true">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>0</height>
          </size>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_8">
          <property name="spacing">
           <number>0</number>
          </property>
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item>
           <widget class="QLabel" name="intelligenceLabel">
            <property name="minimumSize">
             <size>
              <width>30</width>
              <height>20</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>30</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="text">
             <string>智力:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="MYStrip" name="intelligenceStrip" native="true">
            <property name="minimumSize">
             <size>
              <width>120</width>
              <height>20</height>
             </size>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="intelligenceBarNumberLabel">
            <property name="minimumSize">
             <size>
              <width>90</width>
              <height>20</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>90</width>
              <height>19</height>
             </size>
            </property>
            <property name="text">
             <string>999999/999999</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item row="9" column="0" colspan="4">
        <widget class="QGroupBox" name="groupBox">
         <property name="title">
          <string>吉尔</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout">
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_3">
            <item>
             <widget class="QLabel" name="penisEnum">
              <property name="text">
               <string>标签:三个字</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="penisLong">
              <property name="text">
               <string>长度:500MM</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="penisDiameter">
              <property name="text">
               <string>直径:100MM</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_9">
            <item>
             <widget class="QLabel" name="penisCount">
              <property name="text">
               <string>使用次数:9223372036854775807</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="penisSemen">
              <property name="text">
               <string>单次射精量:9223372036854775807ML</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>MYStrip</class>
   <extends>QWidget</extends>
   <header>Windows/MYStrip.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>ColorView</class>
   <extends>QWidget</extends>
   <header>Windows/ColorView.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
